### PR TITLE
Added Kadane Algo

### LINKED ISSUE #100: Implement Kadane Algo
Can this issue be assigned to me?

### PR DESCRIPTION
**Pull request by ivaibhavjindal**
Closes #100

### CODE REVIEW

File: Kadane/kadane.cpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+using namespace std;
+
+int kadmax(int arr[], int size)
+{    
+    int finalMax = INT_MIN, currentMax = 0; //INT_MIN = MOST NEGATIVE INTEGER
+    
+    for(int i = 0; i < size; i++)
+    {
+        //Iterate over Array adding each element to current max
+        currentMax = currentMax + arr[i];
```
Comment (Bhavnaharitsa) [inline Kadane/kadane.cpp]: wrong declaration of variable

File: Kadane/kadane.cpp [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        
+        //finalMax updated when less than currentMax
+        if(finalMax < currentMax)
+            finalMax = currentMax;
```
Comment (Bhavnaharitsa) [inline Kadane/kadane.cpp]: same. This type of method is used in functions

File: Kadane/kadane.cpp [hunk 3]
```diff
@@ -0,0 +1,46 @@
+//MAXIMUM SUM OF SUBARRAY(CONTIGUOUS SUBSEQUENCE) KADANE'S ALGO
+
```
Comment (Bhavnaharitsa) [inline Kadane/kadane.cpp:8]: sample input output

**Comment by Bhavnaharitsa** [comment]
The algorithm doesn't run properly in the first place
image
See this!

**Comment by ivaibhavjindal** [comment]
I've added the Sample input/output and done the snake casing of variable names and camel casing of function name.
Regarding the working of algorithm, Can I know which compiler are you using because I've double checked the code and ran with g++ on my Computer and many different Online Compilers and the code works fine on each.
Codechef
OnlineGDB
CodingGround
g++

**Pull request merged**